subiquity/common/apidef.py

```diff
@@ -274,6 +274,12 @@ def GET() -> bool:
         class has_bitlocker:
             def GET() -> List[Disk]: ...
 
+        class autoinstall_applied:
+            def GET(wait: bool = False) -> None:
+                """Poll or wait for storage autoinstall to be applied to the
+                model."""
+                ...
```

Comment:
obscure but you don't need the ... if you have a docstring :-)

Reply:
Done, thanks.